### PR TITLE
Add cadiz file to class repo

### LINKED ISSUE #23: iservan tracks progress on GitHub Flow
- [] Create a branch **(git)**
- [] Add some commits **(git)**
- [] Open a pull request _**(GitHub)**_
- [] Additional commits **(git)**
- [] merge **(git)**

### PR DESCRIPTION
**Pull request by iservan**
This PR adds a Cadiz file to class repo. This PR closes #23 
- @xiscapericas do u know any good place to eat _ensaimada_ in Mallorca? 🍰 🍡 🐹 ✌️

### CODE REVIEW

File: iservan-cadiz.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+### Restaurants
+
+### Atractions for tourist
+
+- Go to the beach and coastine
+- Visit the old center twon & the cathedral
+
+### Atractions for locals
+
+- Go to the beach
```
Comment (xiscapericas) [inline iservan-cadiz.md]: @iservan : which beach do you recommend? 👙

**Comment by xiscapericas** [comment]
In my hometown! Obviously!! 💟

File: iservan-cadiz.md (hunk 1 above)
Comment (iservan) [inline iservan-cadiz.md]: @xiscapericas if you travel around Cadiz you can't miss **Playa de los Alemanes** in Zahara de los Atunes 🐟 🐠 🌊 ☀️ 😃

**Comment by ruthtubau** [comment]
🚢 :shipit: ⚓

**Comment by iservan** [comment]
Merging my last change

**Pull request closed without merging**